src/spec_tools/visitor.cljc

```diff
@@ -55,7 +68,7 @@
 
 (defmethod visit 'clojure.spec/every [spec accept]
   (let [[_ inner-spec & kwargs] (formize spec)]
-    (accept 'clojure.spec/every spec [(visit inner-spec accept)])))
+    (accept 'clojure.spec/every spec [(visit (++expand-every-cljs-spec-bug++ inner-spec) accept)])))
```

Comment:
Does this work with something like `(s/every (fn [x] (integer? x)))`? If it does, I'm fine with it. 

Did you find the bug on CLJS JIRA? Could you report it if it hasn't been reported yet? I thought it was, but can't find it.